servicetalk-data-protobuf/src/main/java/io/servicetalk/data/protobuf/ProtobufSerializerFactory.java

```diff
@@ -70,4 +88,38 @@ private ProtobufSerializerFactory() {
                 parser2 -> new VarIntLengthStreamingSerializer<>(serializerDeserializer((Parser<T>) parser2),
                         MessageLite::getSerializedSize));
     }
+
+    /**
+     * Get a {@link StreamingSerializerDeserializer} which supports &lt;VarInt length, value&gt; encoding as described
+     * in <a href="https://developers.google.com/protocol-buffers/docs/techniques">Protobuf Streaming</a>.
+     * @param clazz Used to obtain a {@link Parser} which is used to serialize and deserialize.
+     * @param <T> The type to serialize and deserialize.
+     * @return a {@link StreamingSerializerDeserializer} which supports &lt;VarInt length, value&gt; encoding as
+     * described in <a href="https://developers.google.com/protocol-buffers/docs/techniques">Protobuf Streaming</a>.
+     * @see VarIntLengthStreamingSerializer
+     */
+    public <T extends MessageLite> StreamingSerializerDeserializer<T> streamingSerializerDeserializer(Class<T> clazz) {
+        @SuppressWarnings("unchecked")
+        final Parser<T> parser = (Parser<T>) parserMap.get(clazz);
```

Comment:
Perhaps encapsulate the `parserMap.get` in `newParser` and use `computeIfAbsent`?